spec/services/export_activity_to_csv_spec.rb

```diff
@@ -52,7 +52,7 @@
 
   describe "#next_four_quarter_forecasts" do
     it "gets the forecasted total for the date ranges of the next four quarters" do
```

Comment:
I guess this answers my previous question about the date ranges working within the same financial quarter only :-)